src/modules/jackrack/factory.c

```diff
@@ -499,6 +683,30 @@ MLT_REPOSITORY
     }
 #endif
 
+#ifdef WITH_VST2
```

Comment:
And `GPL`

Reply:
this mean I remove the GPL processors conditionals even from the ladspa code that I don't wrote?

Reply:
No, it means this code requires the GPL license via CMake option `GPL`. However, when you make the change I require for `src/modules/jackrack/CMakeLists.txt` then it will only define `WITH_VST2` when `GPL` is on; so, this change is not required.